package.json

```diff
@@ -69,6 +70,7 @@
     "clipanion": "^4.0.0-rc.3",
     "edge-runtime": "^2.5.7",
     "esbuild": "^0.19.11",
+    "form-urlencoded": "^6.1.4",
```

Comment:
do we want to make this a dependency, or should users handle this themselves? what about other response types like `application/xml`, `text/csv`, etc?

Reply:
We need url encoded in requests for dormakaba

For responses yeah we can probably remove it

Reply:
yep makes sense to have it for requests